src/components/Info/_info.scss

```diff
@@ -0,0 +1,48 @@
+.info-section__heading {
+  @include carbon--type-style('heading-01');
+}
+
+.info_card__heading {
```

Comment:
This should be `info-card__heading` - Your headings rare not rendering with the `productive-heading-03` type style.